diff '--color=auto' -urN devoidwm/config.h devoidwm_mod/config.h
--- config.h	2022-01-11 10:52:07.608623111 +0530
+++ mod_config.h	2022-01-11 10:50:06.868624824 +0530
@@ -107,6 +107,9 @@
     {MODKEY|ShiftMask, XK_7, tag, {.ui = 1 << 6}},
     {MODKEY|ShiftMask, XK_8, tag, {.ui = 1 << 7}},
     {MODKEY|ShiftMask, XK_9, tag, {.ui = 1 << 8}},
+
+    {MODKEY, XK_equal, incgap, {.i = 2}},
+    {MODKEY, XK_minus, incgap, {.i = -2}},
 };
 
 #endif
Binary files devoidwm/.git/index and devoidwm_mod/.git/index differ
diff '--color=auto' -urN devoidwm/src/client.c devoidwm_mod/src/client.c
--- src/client.c	2022-01-11 10:52:07.608623111 +0530
+++ mod_src/client.c	2022-01-11 10:50:06.868624824 +0530
@@ -231,3 +231,11 @@
 
     return c.pixel;
 }
+
+void incgap(Arg arg) {
+    if (getfullscrlock(0)) return;
+    int newgap = MAX(border_width + 3, arg.i + gap);
+    if (newgap == gap) return;
+    gap = (unsigned int)newgap;
+    tile();
+}
diff '--color=auto' -urN devoidwm/src/client.h devoidwm_mod/src/client.h
--- src/client.h	2022-01-11 10:52:07.608623111 +0530
+++ mod_src/client.h	2022-01-11 10:50:06.868624824 +0530
@@ -38,5 +38,6 @@
 void lock_fullscr(Client *c);
 void unlock_fullscr(Client *c);
 unsigned int getcolor(const char *color);
+void incgap(Arg arg);
 
 #endif
