--- config.h	2021-10-06 20:35:16.859993199 +0530
+++ patches/config.h	2021-10-06 20:35:09.059993471 +0530
@@ -8,6 +8,9 @@
 static unsigned int margin_left = 6;
 static unsigned int gap = 18; // gap between 2 windows
 
+// lemonaid
+static const int lemonaid_signal = 1;
+
 static const Rule rules[] = {
     /* xprop:
      * WM_CLASS(STRING) = class
--- devoid.c	2021-10-06 20:35:00.696660428 +0530
+++ patches/devoid.c	2021-10-06 20:34:52.509994050 +0530
@@ -631,6 +631,11 @@
     data[0] = ws + 1;
     CHANGEATOMPROP(net_atoms[NetCurrentDesktop], XA_CARDINAL,
                    (unsigned char *)data, 1);
+
+    // lemonaid
+    char command[30];
+    snprintf(command, 30, "pkill -RTMIN+%d lemonaid", lemonaid_signal);
+    system(command);
 }
 
 void apply_window_state(Client *client) {
